_includes/markdown/Web-Vitals.md

```diff
@@ -0,0 +1,293 @@
+
+At 10up, we closely monitor Core Web Vitals (a subset of Web Vitals) during development which in June [2021], will be introduced into Googles ranking algorythm. Ensuring healthy Web Vitals through out build and/or retainers is of paramount importance and requires a shift not only in how we go about building components, but in mainting a high level of quality across overall user experience.
+
+### Cross Discipline Approach
+At 10up we acknowledge that achieving healthy Web Vitals across the board is not siloed to one discipline. Ensuring healthy Web Vitals requires a cross discipline approach spanning Front-end Engineering, Web Engineering, Systems, Audience and Revenue and Design.
+
+As defined by Google, the 3 Core Web Vitals are currently:
+
+* [Largest Contentful Paint](https://web.dev/lcp/)
+* [Cumulative Layout Shift](https://web.dev/cls/)
+* [First Input Delay](https://web.dev/fid/)
+
+<h2 id="lcp" class="anchor-heading">Largest Contentful Paint {% include Util/link_anchor anchor="lcp" %} {% include Util/top %}</h2>
+Largest Contentful Paint is an important metric for measuring pecieved user performance, specifically *loading performance*.
+This metric reports the render time of the largest element on the page _that is visible to the user_.
+
+> An LCP score of 2.5 seconds or less is considered to be a conducive measurement for good user experience.
+
+LCP is measured in seconds (s) and can be tracked against the follow DOM elements:
+
+* `<img>`
+* `<image>` - inside an SVG
+* `<video>`
+* An element with a `background-image`
+* Any element that is considered to be block-level (`display: block`)
+
+### How to diagnose Largest Contentful Paint
+The quickest way to diagnose the Largest Contentful Paint element on the page is by following these steps:
+
+1. Open _Google Chrome_
+2. Open _Chrome DevTools_
+3. Select the _Performance_ Tab
+4. Check the _Web Vitals_ checkbox
+5. Click the _Reload_ button or hit `⌘ ⇧ E` shortcut
+6. Scroll down to _Timings_
+7. Select the green _LCP_ marker
+8. In _Summary_ scroll down to "Related Node"
+9. Click the _node_ listed and it will be highlighted in the _DOM_.
+
+### How to fix Largest Contentful Paint
+
+Once you have diagnosed which element on the page has the Largest Contentful Paint, the next step is to figure out why. 
+There are 3 main factors that contribute to LCP:
+
+1. Slow server response times.
+2. Render-blocking JavaScript and CSS.
+3. Resource load times.
+
+It's important that your server is optimized in a way that doesnt have a domino effect on other vitals. 
+To measure the "speed" of your server you can track the [Time to First Byte (TTFB)](https://web.dev/time-to-first-byte) vital.
+
+Here are some high-level guidelines for ensuring Largest Contentful Paint occurs as fast as possible:
+
+* Are assets (Images, JavaScript, CSS, Video) served over a CDN?
```

Comment:
@dainelmawer 

for consistency, maybe have all as statements rather some as statements and some as questions? 

Example for this line:
* Serve assets (Images, JavaScript, CSS, Video) over a CDN if viable.